ClosedXML/Excel/Cells/XLCells.cs

```diff
@@ -105,12 +120,11 @@ private IEnumerable<XLCell> GetUsedCellsInRange(XLRangeAddress rangeAddress, XLW
             var maxColumn = normalizedAddress.LastAddress.ColumnNumber;
 
             var cellRange = worksheet.Internals.CellsCollection
-                .GetCells(minRow, minColumn, maxRow, maxColumn, _predicate)
-                .Where(c => !c.IsEmpty(_options));
```

Comment:
this was a capturing closure (memory allocations), faster to do inside next loop as extra check